src/dependency-manager/src/service-config/v1.ts

```diff
@@ -356,6 +356,12 @@ export class ServiceConfigV1 extends ServiceConfig {
   @IsOptional({ always: true })
   interfaces?: Dictionary<InterfaceSpecV1>;
 
+  @Type(() => LivenessProbeV1)
+  @IsOptional({ always: true })
+  @IsInstance(LivenessProbeV1, { always: true })
+  @Exclusive(['path', 'command'], { always: true, message: 'Path and command are exclusive.' })
```

Comment:
I like it. Can I suggest we define on each field like oclif?
```ts
{
  @Exclusive(['command'], { always: true })
  path?: string;
  @Exclusive(['command'], { always: true })
  port?: number;
  @Exclusive(['path', 'port'], { always: true })
  command?: string;
}
```

Reply:
Probably worth looking at how ValidateIf gets access to the target obj for checking other fields than the one you're currently validating.